.github/pull_request_template.md

```diff
@@ -15,6 +15,12 @@ cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.
 1. Test steps should be captured in the demo GIF(s) and/or screenshots below.
 > *Demo GIF(s) and screenshots for testing procedure*
 
+## Tabletop
+
+- [ ] Ready for Tabletop (Reviewer check this)
+- [ ] Tabletop scheduled (Bug writer check this)
```

Comment:
what does the `bug writer check this` mean? @riatzukiza

Reply:
Hoping to get a laugh out of 5 minutes of effort and a very easy to review PR :D

Reply:
It means the developer who wrote the ~~bug~~ code in the PR.